packages/lib-classifier/src/store/ClassificationStore.js

```diff
@@ -89,12 +96,17 @@ const ClassificationStore = types
       self.loadingState = asyncStates.success
     }
 
-    function updateClassificationMetadata (newMetadata) {
-      const classification = self.active
-      classification.metadata = Object.assign({}, classification.metadata, newMetadata)
+    function updateClassificationMetadata(newMetadata) {
+      const validClassificationReference = isValidReference(() => self.active)
+      if (validClassificationReference) {
+        const classification = self.active
+        classification.metadata = Object.assign({}, classification.metadata, newMetadata)
+      } else {
+        console.error('No active classification. Cannot update metadata')
```

Comment:
The tests could stub `condole.error` to do nothing.

Reply:
I'd prefer to use `process.browser` check because there were some MST warnings being logged as console errors that although didn't break a test were still essential in figuring out what needed to happen for the upgrade.